Tasks/GooglePlayStatusUpdate/googleutil.ts

```diff
@@ -0,0 +1,256 @@
+    return getTrack.data;
+}
+
+/**
+ * Update a given release track with the given information
+ * Assumes authorized
+ * @param {string} packageName - unique android package name (com.android.etc)
+ * @param {string} track - one of the values {"internal", "alpha", "beta", "production"}
+ * @param {integer or [integers]} versionCode - version code returned from an apk call. will take either a number or a [number]
+ * @param {string} status - one of the values {"draft", "inProgress", "halted", "completed"}
+ * @param {double} userFraction - for rollouting out a release to a track.
+ * @param {releaseNotes} releaseNotes - optional release notes to be attached as part of the update
+ * @returns {Promise} track - A promise that will return result from updating a track
+ *                            { track: string, versionCodes: [integer], userFraction: double }
+ */
+export async function updateTrack(edits: any, packageName: string, track: string, versionCode: any, status: string, userFraction: number, releaseNotes?: ReleaseNotes[]): Promise<Track> {
+    tl.debug('Updating track');
+    const release: AndroidRelease = {
+        versionCodes: (typeof versionCode === 'number' ? [versionCode] : versionCode)
+    };
+
+    if (releaseNotes && releaseNotes.length > 0) {
+        tl.debug('Attaching release notes to the update');
+        release.releaseNotes = releaseNotes;
+    }
+
+    if (!Number.isNaN(userFraction)) {
+        release.userFraction = userFraction;
+    }
+    release.status = status;
+
+    const requestParameters: PackageParams = {
+        packageName: packageName,
+        track: track,
+        resource: {
+            track: track,
+            releases: [release]
+        }
+    };
+
+    tl.debug('Additional Parameters: ' + JSON.stringify(requestParameters));
+    const updatedTrack = await edits.tracks.update(requestParameters);
+    return updatedTrack.data;
+}
+
+/**
+ * Update the universal parameters attached to every request
+ * @param {string} paramName - Name of parameter to add/update
+ * @param {any} value - value to assign to paramName. Any value is admissible.
+ * @returns {void} void
+ */
+export function updateGlobalParams(globalParams: GlobalParams, paramName: string, value: any): void {
+    tl.debug('Updating Global Parameters');
+    tl.debug('SETTING ' + paramName + ' TO ' + JSON.stringify(value));
+    globalParams.params[paramName] = value;
+    google.options(globalParams);
+    tl.debug('Global Params set to ' + JSON.stringify(globalParams));
+}
+
+/**
+ * Adds an apk to an existing edit
+ * Assumes authorized
+ * @param {string} packageName unique android package name (com.android.etc)
+ * @param {string} apkFile path to apk file
+ * @returns {Promise} apk A promise that will return result from uploading an apk
+ *                          { versionCode: integer, binary: { sha1: string } }
+ */
+export async function addApk(edits: any, packageName: string, apkFile: string): Promise<Apk> {
```

Comment:
I don't think we need `addApk` and `uploadDeobfuscation` functions in this task. Could you please remove them and clean up all other unused functions/types?

Reply:
Sure, I have cleaned up all those unused functions/types. Thanks